libs/langchain/langchain/chains/combine_documents/base.py

```diff
@@ -15,19 +16,62 @@
 from langchain.chains.base import Chain
 from langchain.text_splitter import RecursiveCharacterTextSplitter, TextSplitter
 
-DEFAULT_DOCUMENT_SEPARATOR = "\n\n"
+""" --- Constants --- """
```

Comment:
The """ is relatively unconventional, `#` is the norm https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments